Azure/sonic-buildimage
# Export libyang API "lyd_check_mandatory_tree" for Management framework (CVL)

**- Why I did it**
Management framework (CVL) needs to call lyd_check_mandatory_tree() for validation and hence exported lyd_check_mandatory_tree() as an API.

**- How I did it**
Added "API" keyword  before lyd_check_mandatory_tree() definition.

**- How to verify it**
There is no functionality code change here and no specific steps to verify it. Management framework (CVL) should be able to call this function and no patching and compilation error should be seen.

**- Which release branch to backport (provide reason below if selected)**




**- Description for the changelog**

Added "API" keyword before export lyd_check_mandatory_tree() function definition. 

**- A picture of a cute animal (not mandatory but encouraged)**
